pages/topics/all/index.tsx

```diff
@@ -28,9 +29,15 @@ export default function Overview() {
     const cultureTopics:string[] = t("cultureTopics", { returnObjects: true })
     const churchTopics:string[] = t("churchTopics", { returnObjects: true })
 
+    const cultureLinks:string[] = t("cultureLinks", { returnObjects: true })
+    const churchLinks:string[] = t("churchLinks", { returnObjects: true })
+
     const title1 = t("topHeading")
     const title2 = t("botHeading")
 
+    const heroTitle = t("pageTitle")
+    const heroSubtext = t("pageSubtitle").split(".")
```

Comment:
I worry this might cause future issues if Sarah changes the text? Would it be better off / more clear for the editors if we just had it as a list?